packages/synapse-react-client/src/components/FeaturedResearch/FeaturedResearch.tsx

```diff
@@ -0,0 +1,361 @@
+import {
+  Box,
+  CardMedia,
+  Link,
+  Stack,
+  Typography,
+  Skeleton,
+} from '@mui/material'
+import {
+  FileHandleAssociateType,
+  FileHandleAssociation,
+  QueryBundleRequest,
+  Row,
+} from '@sage-bionetworks/synapse-types'
+import { SynapseConstants, SynapseUtilityFunctions } from '../../utils'
```

Comment:
This imports every single utility, which is unnecessary/slow. You can make these imports more specific:
```suggestion
import * as SynapseConstants from '../../utils/SynapseConstants'
import { parseEntityIdFromSqlStatement } from '../../utils/functions/SqlFunctions'
```